Modules/Libs/QuestieLib.lua

```diff
@@ -129,23 +129,29 @@ end
 ---@param showState boolean @ Whether to show (Complete/Failed)
 ---@param blizzLike boolean @True = [40+], false/nil = [40D/R]
 function QuestieLib:GetColoredQuestName(questId, showLevel, showState, blizzLike)
-    local name = QuestieDB.QueryQuestSingle(questId, "name");
+    local quest = QuestieDB:GetQuest(questId)
```

Comment:
What's the reason for this change? `QueryQuestSingle` is many times faster than `QuestieDB:GetQuest` especially if you only need a few values.

Reply:
I'll re-think this